### PR TITLE
USWDS 2.3.1 -- updated

### PR DESCRIPTION
**Pull request by cantsin**
## Description

Examining what is involved with an upgrade to USWDS 2.x. Relevant issue: #867

## Additional information

Most of the work was squashed from https://github.com/18F/tock/pull/945 -- making a concerted effort to finish this off.

### CODE REVIEW

**Comment by cantsin** [comment]
Known issues: 
- Notes displayed doesn't show up on time card form
- Admin dropdown menu changes width when clicked on
- Edit user forms may still need more styling

CCing @hbillings

**Comment by hbillings** [comment]
Fixed the nav! I can't seem to replicate the disappearing notes boxes, and all the edit user forms I can find are in the Django admin, which we don't need to touch. (let me know if they're hiding somewhere else!)

The failing tests seem to be a result of the HTML parser trying to parse the javascript that's included at the bottom of `timecard_form.html`. Removing the JS fixes the parsing issue, but the JS relies on certain Django variables, and I haven't gotten all that refactored yet.

**Comment by tbaxter-18f** [comment]
There are forms on the user pages, like this one: https://tock.18f.gov/employees/e/timothy.baxter/

However, I don't think they're used and I don't think they should be there. The folks who can/should change that can/should do it in the admin.

So I'd like to propose that for traceability we log a separate issue to remove them, and then we can merge that work into this branch and don't have to worry about them anymore.

**Comment by cantsin** [comment]
@hbillings you rock! :tada: Thank you! 

Completely agree that these user forms should be removed if they're not used. If we remove these forms, then we can get rid of some SCSS changes too as they are specific to these forms (particularly `.usa-checkbox` styling changes).

**Comment by tbaxter-18f** [comment]
OK, with #1048 merged, we should now be able to pull master in and resolve a lot of the issues.

File: tock/tock/templates/employees/user_form.html [hunk 1]
```diff
@@ -1,11 +1,10 @@
...
 {% block content %}
-  <h1>Let's Tock about {{ username }}!</h1>
-  {% include 'employees/user_utilization.html' %}
-  {% include 'employees/user_recent_tocks.html' %}
-{% endblock %}
-
+<h1>Let's Tock about {{ username }}!</h1>
+{% include 'employees/user_utilization.html' %}
+{% include 'employees/user_recent_tocks.html' %}
+{% endblock %}
```
Comment (tbaxter-18f) [inline tock/tock/templates/employees/user_form.html:10]: Not looking for any change here, but does it bother anyone else that we don't indent within blocks? Should we have a convention to start doing so, or should I just let it go :frozen:

**Comment by tbaxter-18f** [comment]
I admire the dedication tracking down all the instances of the changed classname. I probably would have just subclassed the new name to the old name and called it good. :-D

File: tock/tock/templates/employees/user_form.html (hunk 1 above)
Comment (hbillings) [reply tock/tock/templates/employees/user_form.html:10]: Sorry, this is probably my bad -- VSCode sometimes automatically un-indents things and it drives me crazy but I can't figure out how to turn it off 😬

Comment (cryptofilegsa) [reply tock/tock/templates/employees/user_form.html:10]: possibly this setting in VSCode editor settings?  
image

**Pull request merged**